src/constants/cctp.ts

```diff
@@ -1,11 +1,44 @@
 import cctpTokens from '../../public/configs/cctp.json';
+import { TransferType, TransferTypeType } from './abacus';
 
-const CCTP_MAINNET_CHAINS = cctpTokens.filter((token) => !token.isTestnet);
-const CCTP_MAINNET_CHAINS_NAMES_LOWER_CASE = CCTP_MAINNET_CHAINS.map((token) =>
-  token.name.toLowerCase()
-);
+export type CctpTokenInfo = {
+  chainId: string;
+  tokenAddress: string;
+  name: string;
+};
 
-// TODO: make a general capitalize util fn
-export const CCTP_MAINNET_CHAIN_NAMES_CAPITALIZED = CCTP_MAINNET_CHAINS_NAMES_LOWER_CASE.map(
-  (tokenName) => tokenName[0].toUpperCase() + tokenName.slice(1)
-);
+type NullableTransferType = TransferTypeType | undefined | null;
+
+// Ethereum is a high fee chain for withdrawals but not deposits
+const getLowestFeeChains = (type: NullableTransferType) =>
+  type === TransferType.deposit ? cctpTokens : cctpTokens.filter(({ chainId }) => chainId !== '1');
+
+// move this out if we need it in another module
```

Comment:
opted to not move it out in this PR since theres no other usecase for it. can replace it with lodash if ppl feel strongly